web/src/settings_components.js

```diff
@@ -476,7 +480,10 @@ export function get_auth_method_list_data() {
 }
 
 export function parse_time_limit($elem) {
-    return Math.floor(Number.parseFloat(Number($elem.val())).toFixed(1) * 60);
+    if (Number.isNaN(Number($elem.val()))) {
```

Comment:
This check is needed to make sure we do not consider values like "24a" as valid, `Number.parseInt` and `Number.parseFloat` would convert it to `24`.

Reply:
Let's put this in a comment, since I think otherwise someone is going to be tempted to simplify. Might even be worth just extracting this bit of logic as a helper that accepts `$elem.val` as an input and returns a `number | NaN`.

Reply:
Updated to extract a helper function.